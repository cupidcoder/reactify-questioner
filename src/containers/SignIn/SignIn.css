.SignIn {
  min-height: 80vh;
}
