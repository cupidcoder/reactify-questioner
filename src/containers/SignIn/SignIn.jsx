import React from 'react';
import SignInForm from '../../components/SignInForm/SignInForm';
import style from './SignIn.css';

const SignIn = () => (
  <div className={style.SignIn}>
    <SignInForm />
  </div>
);

export default SignIn;
