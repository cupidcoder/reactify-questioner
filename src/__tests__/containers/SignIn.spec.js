import React from 'react';
import { shallow } from 'enzyme';
import SignIn from '../../containers/SignIn/SignIn';

describe('SignIn', () => {
  it('Should correctly save snapshot of the SignIn page', () => {
    const wrapper = shallow(<SignIn />);
    expect(wrapper).toMatchSnapshot();
  });
});
