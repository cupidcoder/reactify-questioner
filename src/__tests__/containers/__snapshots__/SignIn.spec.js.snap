// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn Should correctly save snapshot of the SignIn page 1`] = `ShallowWrapper {}`;
